### PR TITLE
Databinding + bugfix

### PR DESCRIPTION
**Pull request by Revenaunt**
Partially transposed Launcher and PackageEnrollment Activities to databinding and fix package enrollment notification bug

### CODE REVIEW

File: src/main/java/org/tndata/android/compass/fragment/LauncherFragment.java [hunk 1]
```diff
@@ -9,9 +10,9 @@
 import android.view.View;
 import android.view.View.OnClickListener;
 import android.view.ViewGroup;
-import android.widget.ProgressBar;
 
 import org.tndata.android.compass.R;
+import org.tndata.android.compass.databinding.FragmentLauncherBinding;
```
Comment (bradmontgomery) [inline src/main/java/org/tndata/android/compass/fragment/LauncherFragment.java]: Did you check this in?  Seems to be missing from the repo.

Comment (Revenaunt) [inline src/main/java/org/tndata/android/compass/fragment/LauncherFragment.java]: No, those are generated, you need to restart the IDE

Comment (bradmontgomery) [inline src/main/java/org/tndata/android/compass/fragment/LauncherFragment.java]: oh, gotcha. My android Studio was freaking out! 💥

Comment (Revenaunt) [inline src/main/java/org/tndata/android/compass/fragment/LauncherFragment.java]: Heh, mine as well when I got started, and it is funny, every time I make a layout databinding compatible I gotta restart the IDE, rebuilding doesn't do the trick...

**Pull request merged**